project-base/storefront/components/Layout/Footer/FooterPlaceholder.tsx

```diff
@@ -0,0 +1,57 @@
+import { FooterProps } from './Footer';
+import imageLogo from '/public/images/logo.svg';
+import { ExtendedNextLink } from 'components/Basic/ExtendedNextLink/ExtendedNextLink';
+import { Image } from 'components/Basic/Image/Image';
+import { useDomainConfig } from 'components/providers/DomainConfigProvider';
+import useTranslation from 'next-translate/useTranslation';
+import { getInternationalizedStaticUrls } from 'utils/staticUrls/getInternationalizedStaticUrls';
+
+export const FooterPlaceholder: FC<FooterProps> = ({ simpleFooter, footerArticles, opening, phone }) => {
+    const { t } = useTranslation();
+    const { url } = useDomainConfig();
+    const [cookieConsentUrl, contactUrl] = getInternationalizedStaticUrls(['/cookie-consent', '/contact'], url);
+
+    return (
+        <>
+            {!simpleFooter && (
+                <>
+                    <a className=" text-white" href={'tel:' + phone}>
+                        {phone}
+                    </a>
+                    <p>{opening}</p>
+                    <ExtendedNextLink href={contactUrl}>{t('Write to us')}</ExtendedNextLink>
+                    {footerArticles?.map((item) => (
+                        <div key={item.title}>
+                            <h3 className="text-white">{item.title}</h3>
+                            <ul>
+                                {item.items.map((item) => (
+                                    <li key={item.uuid}>
+                                        <ExtendedNextLink
+                                            className="text-white"
+                                            href={item.__typename === 'ArticleSite' ? item.slug : item.url}
+                                            rel={item.external ? 'nofollow noreferrer noopener' : undefined}
+                                            target={item.external ? '_blank' : undefined}
+                                        >
+                                            {item.name}
+                                        </ExtendedNextLink>
+                                    </li>
+                                ))}
+                            </ul>
+                        </div>
+                    ))}
+                </>
+            )}
+            {t('Copyright © 2021, Shopsys s.r.o. All rights reserved.')}
```

Comment:
off topic - we still dont have dynamic year in footer txt?

Reply:
Nope. Not even the phone number and opening hours...